src/Type/Object/User.php

```diff
@@ -92,6 +82,22 @@ public static function register_type() {
 						'type'        => 'String',
 						'description' => __( 'The nicename for the user. This field is equivalent to WP_User->user_nicename', 'wp-graphql' ),
 					],
+					'status'          => [
+						'type'        => 'String',
+						'description' => __( 'The status for the user. This field is equivalent to WP_User->user_status', 'wp-graphql' ),
+					],
+					'activation_key'          => [
+						'type'        => 'String',
+						'description' => __( 'The activation_key for the user. This field is equivalent to WP_User->user_activation_key', 'wp-graphql' ),
+					],
+					'pass'          => [
```

Comment:
`Pass` is not descriptive to a consumer. It's an abbreviation, but the word "Pass" also has many meanings. We should consider being more explicit and name this field `password`.

Reply:
I agree. However, I was distracted by the way WP names this in the WP_user class. Do we move away from that?

Reply:
ya, so WordPress internals are `snake_case` but GraphQL uses `camelCase` for field names and `PascalCase` for Type names, so at the layers that define the GraphQL schema, we are using `camelCase` for fields. 

We don't want to enforce this centrally. If you _do_ want to use other cases in your field names for custom plugins, etc, you can, but in general we want to follow common practices in the greater GraphQL ecosystem.